server/controllers/personalInfo.js

```diff
@@ -0,0 +1,21 @@
+const PersonalInfo = require('../models/PersonalInfo');
+const asyncHandler = require("express-async-handler");
+
+exports.createPersonalInfo = asyncHandler(async (req, res) => {
+  try {
+    const info = PersonalInfo.create({
+      userId: req.user.id,
+      firstName: req.body.firstName,
```

Comment:
As almost all the fields names are similar here, you spread the `req.body` to make it look a lot cleaner.